### PR TITLE
Code formatting for string 

### PR DESCRIPTION
**Pull request by heljhumenad**
Hi @crazyguitar  i just add another pr because honestly i don't know how to make changes in PR my apology

### CODE REVIEW

**Comment by crazyguitar** [comment]
Hi @heljhumenad 

👍! By the way, could you squash commits from ac6661d to HEAD (35baadd)? thanks

File: docs/notes/python-unicode.rst [hunk 1]
```diff
@@ -240,3 +240,27 @@ Python.
...
+.. code-block:: python
+
+    # Using ``%`` operator
+    >>> print("This is a %s %s" % ('long', 'string'))
+    >>> 'This is a long string' 
+
+    # using new ``format`` function with positional argument
+    >>> print("This is a {0} {1}".format(long, string))
+
+    # Using ``f-string``
```
Comment (zhongjiajie) [inline docs/notes/python-unicode.rst]: Should we notice user `f-string` work only after Python 3.6?
```suggestion
    # Using ``f-string`` new in Python 3.6
```

File: docs/notes/python-unicode.rst [hunk 2]
```diff
@@ -240,3 +240,27 @@ Python.
...
+-----------------
+
+.. code-block:: python
+
+    # Using ``%`` operator
+    >>> print("This is a %s %s" % ('long', 'string'))
+    >>> 'This is a long string' 
+
+    # using new ``format`` function with positional argument
+    >>> print("This is a {0} {1}".format(long, string))
```
Comment (zhongjiajie) [inline docs/notes/python-unicode.rst]: Maybe you forget add quotes to values
```suggestion
    >>> print("This is a {0} {1}".format('long', 'string'))
```

File: docs/notes/python-unicode.rst [hunk 3]
```diff
@@ -240,3 +240,27 @@ Python.
...
+String Formatting 
+-----------------
+
+.. code-block:: python
+
+    # Using ``%`` operator
+    >>> print("This is a %s %s" % ('long', 'string'))
+    >>> 'This is a long string' 
+
+    # using new ``format`` function with positional argument
```
Comment (zhongjiajie) [inline docs/notes/python-unicode.rst]:
```suggestion
    # Using new ``format`` function with positional argument
```

Comment (zhongjiajie) [reply docs/notes/python-unicode.rst]: And should we add `format` with variable names
```py
print("This is a {long} {string}".format(long='long', string='string'))
```

**Comment by crazyguitar** [comment]
Thanks, guys! I do a filter branch for moving some static files to LFS. pls resent PR if you're still interested in this project.

**Pull request closed without merging**